app/models/audit/FactCheckCompleteEvent.scala

```diff
@@ -16,27 +16,31 @@
 
 package models.audit
 
-import org.joda.time.format.DateTimeFormat
 import org.joda.time.LocalDate
+import org.joda.time.format.{DateTimeFormat, DateTimeFormatter}
 import play.api.libs.json.{JsValue, Json, Writes}
 import utils.JsonObjectSugar
 
-case class FactCheckSubmissionEvent(PID: String, processID: String, processTitle: String) extends AuditEvent {
+case class FactCheckCompleteEvent(auditInfo: AuditInfo) extends AuditEvent {
   val submittedDate: LocalDate = LocalDate.now
-  override val transactionName: String = "submittedForFactCheck"
+  override val transactionName: String = "factCheckComplete"
```

Comment:
There seems to be some duplication with transactionName and auditType do we really need both?

Reply:
Have refactored in a separate branch - will incorporate based on comments on that